tools-v2/pkg/cli/command/curvebs/status/etcd/etcd.go

```diff
@@ -68,12 +68,13 @@ func NewEtcdCommand() *cobra.Command {
 }
 
 func (eCmd *EtcdCommand) AddFlags() {
-	config.AddBsEtcdAddrFlag(eCmd.Cmd)
 	config.AddHttpTimeoutFlag(eCmd.Cmd)
+	config.AddBsEtcdAddrFlag(eCmd.Cmd)
```

Comment:
no need

Reply:
> no need

Does the order of add config impact code logic?
I think this order is more good-looking...

Reply:
done